src/components/TrendChip.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <app-chip
+    :variant="priceChipVariant">
+    {{ priceChangeSign }}{{ formatNullable(delta) }} %
+  </app-chip>
+</template>
+<script setup>
+const props = defineProps({
+  delta: {
+    type: Number,
+    required: true,
+  },
+})
+
+const priceChangeSign = computed(() => {
+  return props.delta > 0 ? '+' : ''
```

Comment:
```suggestion
  if (props.delta < 0) {
    return '-'
  } else if (props.delta > 0) {
    return '+'
  } else {
    return ''
  }
```

To cover all the cases, IMO "-" should also be displayed.

Reply:
But this would produce the double minus signing I just fixed. You want it this way?

Reply:
You are right, It would be "-" in any case.